### ISSUE TITLE
`minLength` and `maxLength` on `textarea` field causes payload config validation to fail on startup.

### ISSUE THREAD

**Issue by richardvanbergen** [payloadcms/payload#11]
I hope it's OK to just open issues like this but I _think_ this is a bug so you should _hopefully_ find it useful. 😊

---

### Update (Original below)

Problem is actually down to `minLength` and `maxLength` properties on `textarea`. They don't appear to be expected by whatever code is checking the payload config on startup.

### Reproduction steps

1. New project created with `yarn create payload-app`
2. Create `textarea` with `minLength` or `maxLength` properties.
3. Make sure server isn't running.
4. Start the server and see error.

---

### Expected Behavior

Adding the following config (mostly copied from the documentation) should show a collection with a 'title' and 'description' field.

```
const Page: CollectionConfig = {
  slug: 'page',
  labels: {
    singular: 'Page',
    plural: 'Pages',
  },
  access: {
    create: () => true,
    read: () => true,
    update: () => true,
    delete: () => true,
  },
  fields: [
    {
      name: 'meta', // required
      type: 'group', // required
      label: 'Page Meta',
      fields: [ // required
        {
          name: 'title',
          type: 'text',
          required: true,
          minLength: 20,
          maxLength: 100,
        },
        {
          name: 'description',
          type: 'textarea',
          required: true,
          minLength: 40,
          maxLength: 160,
        }
      ],
    }
  ]
}
```

### Current Behavior

Payload fails to start at payload configuration step due to schema validation failing.

<img width="1068" alt="image" src="https://user-images.githubusercontent.com/3854592/104828824-88f24800-58b0-11eb-8677-ad9566f2aa5a.png">

**BUT! (and this is why I think it's a bug)**

If you comment out the fields, start the server and then uncomment again then it displays as expected.

<img width="846" alt="image" src="https://user-images.githubusercontent.com/3854592/104828890-411ff080-58b1-11eb-86a1-acf0bdd3f289.png">

### Reproduction steps

1. New project created with `yarn create payload-app`
2. Add pages schema as described above.
3. Make sure server isn't running.
4. Start the server and see error.

**Comment by denolfe** [comment]
Hey @richardvanbergen, we appreciate the bug report! It looks like `minLength` and `maxLength` fields are missing as allowed values for `textarea`. I'll get this fixed 👍

**Comment by richardvanbergen** [comment]
I was going to open a pull request but you got there first. Well done on the quick response! 😉

**Comment by denolfe** [comment]
Fixed with 2c98087c

**Comment by denolfe** [comment]
Fix pushed in version 0.1.145 and updated create-payload-app's payload version as well.

@richardvanbergen You can either update your package.json version or any new app w/ create-payload-app will have that version 👍
